### DISCUSSION TITLE
4mmc versus Mephedrone | Naming

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#15] [Classification]
I'd like to discuss the naming of 4mmc versus Mephedrone for the tripsit chart. I do not swing either way at the moment and am opening this for conversation.

**Comment by jedrek8733** [comment]
I'm all for 4mmc. SIN's chart has got it named "4mmc and analogues" with refs on the bottom, I think that would also be a wise move

**Reply by utaninja** [reply]
We discussed adding a note like that. The issue is that may run into problems that we are looking at with amphetamines. As you noted in the other discussion, some amphetamines are SRA's, we may find that certain 4mmc analogs will have pharmacological interactions that were not expected. There are still "new" analogs that are coming out to this day, last ones I've heard of are 3cmc, 2mmc, and 4cma. As noting with 4cma, we don't want to state blanket interactions for drugs that have not been researched.

**Comment by utaninja** [comment]
Related discussions, please use these instead:
[GitHub Pull Request #127](https://github.com/TripSit/drugs/pull/127)
[GitHub Discussion #16](https://github.com/TripSit/drugs/discussions/16)
Shifting from drug categories (e.g., opioids) and wildcard drugs (e.g., 2C-X) to individual classifications. The posts also discuss identifying and categorizing MAOIs within the database and distinguishing NBOH from NBOMe. Additionally, we need to determine if interactions for one drug (e.g., Amphetamines) can be applied to similar stimulants in the database or if each interaction should be reviewed manually.

(Thanks for the help @jedrek8733!)
